scripts/wallet-sanity.sh

```diff
@@ -39,7 +39,7 @@ check_balance_output() {
 
 pay_and_confirm() {
   exec 42>&1
-  signature=$($solana_wallet "${entrypoint[@]}" pay "$@" | tee >(cat - >&42))
+  signature=$($solana_wallet "${entrypoint[@]}" pay "$@" | tail -c 88 | tee >(cat - >&42))
```

Comment:
🙈

Reply:
I know..... 🤢
We could rework the general syntax of wallet responses, if making them nicely machine-parsable is or becomes a priority.

Reply:
Adding a flag for machine parsable seems like a good approach but yeah we'll do that when this next breaks :)